ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -291,6 +401,53 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Projection Test"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "ImportStorage.DefaultProjectionCount = 5;",
```

Comment:
this const should be in the storage

Reply:
Davide, you certainly meant that this const should NOT be in the storage. I moved it to constants notebook at the ifrs17, made a static variable. So it is resolved.